### ISSUE TITLE
Request: Make "Loaded" bigger in the Nimbus select screen.

### ISSUE THREAD

**Issue by guusbuk** [PretendoNetwork/nimbus#7]
![image](https://user-images.githubusercontent.com/78563005/154296724-d5cab5e5-bd1c-40d0-8679-2c9025fdaef6.png)
![image](https://user-images.githubusercontent.com/78563005/154296787-02ca0601-f6d9-4564-953b-a52d84499da5.png)
Can you make the `loaded` part a bit bigger, so it is easier to read on the small 3ds screen?

**Comment by guusbuk** [comment]
Might still be something to consider

**Comment by Golem642** [comment]
I have a question also : why is the Nintendo network logo like that and not the one we see on the boxes and stuff ?

**Comment by kry008** [comment]
> I have a question also : why is the Nintendo network logo like that and not the one we see on the boxes and stuff ?

Due to copyrights

**Comment by jonbarrow** [comment]
Sorry to necro-post on such an old issue, I decided to go through this repo and try to clean things up a bit.

I agree with this, I noticed how small this text is too and it's very hard to read. It might be worth considering looking into moving away from images in general for text like that and doing it in code? Idk, @DaniElectra thoughts?

> I have a question also : why is the Nintendo network logo like that and not the one we see on the boxes and stuff ?

It's supposed to be an eyeball, it was sort of a jab at Nintendo at the time for being "big brother" and watching you

**Comment by DaniElectra** [comment]
Agreed! @jonbarrow Do we have the original project files for the buttons so we can faithfully recreate it in code?

**Comment by jonbarrow** [comment]
Unfortunately I don't think we do. If I remember correctly @shutterbug2000 made these originally years ago, and I believe he's moved machines several times since then. I wouldn't be surprised if they're gone by now
